Flowchart/obj/Debug/netcoreapp1.0/FlowchartCreator.csproj.FileListAbsolute.txt

```diff
@@ -8,3 +8,23 @@ C:\Users\Taylor\Source\Repos\Flowchart-Creator-production\Flowchart\obj\Debug\ne
 C:\Users\Taylor\Source\Repos\Flowchart-Creator-production\Flowchart\obj\Debug\netcoreapp1.0\FlowchartCreator.pdb
 C:\Users\Taylor\Source\Repos\Flowchart-Creator-production\Flowchart\obj\Debug\netcoreapp1.0\UserSecretsAssemblyInfo.cs
 C:\Users\Taylor\Source\Repos\Flowchart-Creator-production\Flowchart\obj\Debug\netcoreapp1.0\FlowchartCreator.csprojResolveAssemblyReference.cache
+C:\Users\Sam\Desktop\Flowchart-Creator-production\Flowchart\bin\Debug\netcoreapp1.0\FlowchartCreator.deps.json
```

Comment:
I don't have it in front of me but you could just try deleting it and running the project. My guess is that it will be generated if it doesn't already exist. I never used to have this prior to .NET Core.